### PR TITLE
Traslate `library/array.po`

### LINKED ISSUE #139: Translate `library/array.po`

### PR DESCRIPTION
**Pull request by tomatoprinx**
resolves python/python-docs-zh-tw#139

### CODE REVIEW

File: library/array.po [hunk 1]
```diff
@@ -34,9 +35,9 @@ msgid ""
 "following type codes are defined:"
 msgstr ""
 "這個模組定義了一個物件型別，可以簡潔的表達一個包含基本數值的陣列：字元、整"
-"數、浮點數。陣列是一個非常類似 list 的序列型態，除了陣列會限制儲存的物件型"
-"別。在建立陣列時可以使用一個字元的  :dfn:`type code` 來指定儲存的資料型別。下"
-"面是 type codes 的定義。"
+"數、浮點數。陣列是一個非常類似 list（串列） 的序列型別，除了陣列會限制儲存的"
+"物件型別。在建立陣列時可以使用一個字元的  :dfn:`type code` 來指定儲存的資料型"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"物件型別。在建立陣列時可以使用一個字元的 :dfn:`type code` 來指定儲存的資料型"
```

File: library/array.po [hunk 2]
```diff
@@ -204,14 +205,17 @@ msgstr "註解："
 
 #: ../../library/array.rst:51
 msgid "It can be 16 bits or 32 bits depending on the platform."
-msgstr ""
+msgstr "根據平台的不同，他有可能是 16 位元 或者 32 位元。"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
msgstr "根據平台的不同，他有可能是 16 位元或者 32 位元。"
```

File: library/array.po [hunk 3]
```diff
@@ -204,14 +205,17 @@ msgstr "註解："
...
 
 #: ../../library/array.rst:53
 msgid ""
 "``array('u')`` now uses ``wchar_t`` as C type instead of deprecated "
 "``Py_UNICODE``. This change doesn't affect to its behavior because "
 "``Py_UNICODE`` is alias of ``wchar_t`` since Python 3.3."
 msgstr ""
+"目前 ``array(‘u’)`` 使用 ``wchar_t`` 取代已棄用的 ``Py_UNICODE`` 作為 C "
+"type。 這個異動並沒有影響到他的作用，因爲自從 Python 3.3 開始 ``Py_UNICODE`` "
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"type。這個異動並沒有影響到他的作用，因爲自從 Python 3.3 開始 ``Py_UNICODE`` "
```

File: library/array.po [hunk 4]
```diff
@@ -233,8 +237,8 @@ msgid ""
 "object`, or iterable over elements of the appropriate type."
 msgstr ""
 "一個新的陣列中的元素被 *typecode* 限制，並由選用的 *initializer* 參數初始"
-"化， *initializer* 必須是一個 list、 :term:`bytes-like object` 或包含適當型別"
-"變數的 iterable 。"
+"化， *initializer* 必須是一個 list（串列）、 :term:`bytes-like object` 或包含"
```
Comment (mattwang44) [inline library/array.po]: You only need to denote the translated content once in an article.
```suggestion
"化， *initializer* 必須是一個 list、\\ :term:`bytes-like object` 或包含"
```

Comment (mattwang44) [inline library/array.po:240]:
```suggestion
"化， *initializer* 必須是一個 list（串列）、\ :term:`bytes-like object` 或包含"
```

File: library/array.po [hunk 5]
```diff
@@ -243,19 +247,21 @@ msgid ""
 "below) to add initial items to the array.  Otherwise, the iterable "
 "initializer is passed to the :meth:`extend` method."
 msgstr ""
-"如果指定一個 list 或 string ，新的陣列初始化時會傳入 :meth:`fromlist` 、 :"
-"meth:`frombytes` 或 :meth:`fromunicode` 方法（參照下方）將元素新增到其中。其"
-"他型態的變數則會傳入 :meth:`extend` 方法初始化。"
+"如果指定一個 list（串列）或 string ，新的陣列初始化時會傳入 :meth:"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"如果指定一個 list 或 string ，新的陣列初始化時會傳入 :meth:"
```

Comment (mattwang44) [inline library/array.po]:
```suggestion
"如果指定一個 list（串列）或 string，新的陣列初始化時會傳入 :meth:"
```

File: library/array.po [hunk 6]
```diff
@@ -243,19 +247,21 @@ msgid ""
...
+"`fromlist` 、 :meth:`frombytes` 或 :meth:`fromunicode` 方法（參照下方）將元素"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"`fromlist`\ 、\ :meth:`frombytes` 或 :meth:`fromunicode` 方法（參照下方）將元素"
```

File: library/array.po [hunk 7]
```diff
@@ -243,19 +247,21 @@ msgid ""
...
+"新增到其中。其他型態的變數則會傳入 :meth:`extend` 方法初始化。"
 
 #: ../../library/array.rst:80
 msgid ""
 "Raises an :ref:`auditing event <auditing>` ``array.__new__`` with arguments "
 "``typecode``, ``initializer``."
 msgstr ""
+"引發 :ref:`審核事件(auditing event) <auditing>` ``array.__new__`` 並帶入 "
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"引發\ :ref:`審核事件 (auditing event) <auditing>` ``array.__new__`` 並帶入引數 "
```

File: library/array.po [hunk 8]
```diff
@@ -243,19 +247,21 @@ msgid ""
...
+"``typecode``, ``initializer``."
```
Comment (mattwang44) [inline library/array.po:260]: Replace with Chinese punctuation.

File: library/array.po [hunk 9]
```diff
@@ -266,6 +272,11 @@ msgid ""
 "interface, and may be used wherever :term:`bytes-like objects <bytes-like "
 "object>` are supported."
 msgstr ""
+"陣列支援常見的序列操作，包含索引（indexing）、切片（slicing）、串接"
```
Comment (mattwang44) [inline library/array.po]: Also update the following line
```suggestion
"陣列支援常見的序列操作，包含索引 (indexing)、切片 (slicing)、串接"
```

File: library/array.po [hunk 10]
```diff
@@ -266,6 +272,11 @@ msgid ""
...
+"（concatenation）、相乘（multiplication）等。當使用切片進行賦值時，賦值的陣列"
+"必須具備相同的 type code ，其他型別的數值將導致  :exc:`TypeError` 。陣列同時"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"必須具備相同的 type code，其他型別的數值將導致 :exc:`TypeError`\ 。陣列同時"
```

File: library/array.po [hunk 11]
```diff
@@ -335,9 +346,9 @@ msgid ""
 "iterable and its elements must be the right type to be appended to the array."
 msgstr ""
 "從 *iterable* 中新增元素到陣列的尾端，如果 *iterable* 是另一個陣列，他必須有"
-"完全相同的 type code ，如果不同會產生 :exc:`TypeError` 。如果 *iterable* 不是"
+"完全相同的 type code ，如果不同會導致 :exc:`TypeError` 。如果 *iterable* 不是"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"完全相同的 type code，如果不同會導致 :exc:`TypeError`。如果 *iterable* 不是"
```

File: library/array.po [hunk 12]
```diff
@@ -360,16 +371,16 @@ msgid ""
 "inserted into the array."
 msgstr ""
 "從 :term:`file object` *f* 讀取 *n* 個 machine value 類型的元素，接著將這些元"
-"素加入陣列的最尾端。如果只有少於 *n* 個有效的元素會產生 :exc:`EOFError` 錯"
-"誤，但有效的元素仍然會被加入陣列中。"
+"素加入陣列的最尾端。如果只有少於 *n* 個有效的元素會導致 :exc:`EOFError` ，但"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"素加入陣列的最尾端。如果只有少於 *n* 個有效的元素會導致 :exc:`EOFError`，但"
```

File: library/array.po [hunk 13]
```diff
@@ -360,16 +371,16 @@ msgid ""
...
+"有效的元素仍然會被加入陣列中。"
 
 #: ../../library/array.rst:168
 msgid ""
 "Append items from the list.  This is equivalent to ``for x in list: a."
 "append(x)`` except that if there is a type error, the array is unchanged."
 msgstr ""
-"從 list 中新增元素。這等價於 ``for x in list: a.append(x)`` ，除了有型態錯誤"
-"產生時，陣列會保持原狀不會被更改。"
+"從 list（串列）中新增元素。這等價於 ``for x in list: a.append(x)`` ，除了有型"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"從 list 中新增元素。這等價於 ``for x in list: a.append(x)``，除了有型"
```

File: library/array.po [hunk 14]
```diff
@@ -389,10 +400,13 @@ msgid ""
 "specified to search for *x* within a subsection of the array.  Raise :exc:"
 "`ValueError` if *x* is not found."
 msgstr ""
+"回傳 *i* 的最小數值，使得 *i* 成為陣列之中第一次出現 *x* 的索引。選擇性的參"
+"數 *start* 及 *stop* 則可以被用來在指定的陣列空間中搜尋 *x* 。如果 *x* 不存在"
+"將導致 :exc:`ValueError` 。"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"將導致 :exc:`ValueError`。"
```

File: library/array.po [hunk 15]
```diff
@@ -389,10 +400,13 @@ msgid ""
 "specified to search for *x* within a subsection of the array.  Raise :exc:"
 "`ValueError` if *x* is not found."
 msgstr ""
+"回傳 *i* 的最小數值，使得 *i* 成為陣列之中第一次出現 *x* 的索引。選擇性的參"
+"數 *start* 及 *stop* 則可以被用來在指定的陣列空間中搜尋 *x* 。如果 *x* 不存在"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"數 *start* 及 *stop* 則可以被用來在指定的陣列空間中搜尋 *x*。如果 *x* 不存在"
```

File: library/array.po [hunk 16]
```diff
@@ -389,10 +400,13 @@ msgid ""
...
+"將導致 :exc:`ValueError` 。"
 
 #: ../../library/array.rst:187
 msgid "Added optional *start* and *stop* parameters."
-msgstr ""
+msgstr "新增選擇性的參數 *start* 及 *stop* 。"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
msgstr "新增選擇性的參數 *start* 及 *stop*。"
```

File: library/array.po [hunk 17]
```diff
@@ -436,16 +450,16 @@ msgstr "將所有元素 （以 machine code 的形式）寫入 :term:`file objec
 
 #: ../../library/array.rst:230
 msgid "Convert the array to an ordinary list with the same items."
-msgstr "不更改元素，將陣列轉為一般的 list 。"
+msgstr "不更改元素，將陣列轉為一般的 list（串列）。"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
msgstr "不更改元素，將陣列轉為一般的 list。"
```

File: library/array.po [hunk 18]
```diff
@@ -458,6 +472,13 @@ msgid ""
 "`eval`, so long as the :class:`~array.array` class has been imported using "
 "``from array import array``. Examples::"
 msgstr ""
+"當一個陣列物件被列印或轉換成字串時，他會被表示為 ``array(typecode, "
+"initializer)`` 。若為空陣列則參數 *initializer* 被省略，若  *typecode* 是 "
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"initializer)``。若為空陣列則參數 *initializer* 被省略，若 *typecode* 是 "
```

**Review by mattwang44 — CHANGES_REQUESTED** [review]
Thanks for the hard work!
Please read through the README for how to correctly use spaces and punctuations. I'll keep reviewing the translated content after the commit for modifying punctuations is pushed.

**Comment by tomatoprinx** [comment]
Hi @mattwang44 
Thanks for your previous review and advice for space and punctuation placement. I had pushed a new commit 3f6f381 to correct it.

File: library/array.po [hunk 19]
```diff
@@ -34,9 +37,9 @@ msgid ""
 "following type codes are defined:"
 msgstr ""
 "這個模組定義了一個物件型別，可以簡潔的表達一個包含基本數值的陣列：字元、整"
-"數、浮點數。陣列是一個非常類似 list 的序列型態，除了陣列會限制儲存的物件型"
-"別。在建立陣列時可以使用一個字元的  :dfn:`type code` 來指定儲存的資料型別。下"
-"面是 type codes 的定義。"
+"數、浮點數。陣列是一個非常類似 list（串列）的序列型別，除了陣列會限制儲存的物"
+"件型別。在建立陣列時可以使用一個字元的 :dfn:`type code` 來指定儲存的資料型"
+"別。下面是 type codes 的定義。"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"別。下面是 type codes 的定義："
```

Comment (mattwang44) [inline library/array.po]:
```suggestion
"別。以下為有被定義的 type codes。"
```

File: library/array.po [hunk 20]
```diff
@@ -252,10 +258,12 @@ msgid ""
 "Raises an :ref:`auditing event <auditing>` ``array.__new__`` with arguments "
 "``typecode``, ``initializer``."
 msgstr ""
+"引發 :ref:`審核事件 (auditing event) <auditing>` ``array.__new__`` 並帶入引"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"引發\\ :ref:`審核事件 (auditing event) <auditing>` ``array.__new__`` 並帶入引"
```

File: library/array.po [hunk 21]
```diff
@@ -252,10 +258,12 @@ msgid ""
...
+"數 ``typecode``， ``initializer``。"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"數 ``typecode``、``initializer``。"
```

File: library/array.po [hunk 22]
```diff
@@ -233,8 +239,8 @@ msgid ""
 "object`, or iterable over elements of the appropriate type."
 msgstr ""
 "一個新的陣列中的元素被 *typecode* 限制，並由選用的 *initializer* 參數初始"
-"化， *initializer* 必須是一個 list、 :term:`bytes-like object` 或包含適當型別"
-"變數的 iterable 。"
+"化，\\ *initializer* 必須是一個 list、\\ :term:`bytes-like object` 或包含適當"
```
Comment (mattwang44) [inline library/array.po]: consider adding the translated content for the term `bytes-like objects`（`類位元組串物件`, as mentioned in the glossary page）.

File: library/array.po [hunk 23]
```diff
@@ -295,7 +308,7 @@ msgid ""
 "it."
 msgstr ""
 "回傳一個 tuple ``(address, length)`` 表示當前的記憶體位置和陣列儲存元素的緩衝"
-"區記憶體長度。緩衝區的長度單位是 bytes ，並可以用 ``array.buffer_info()[1] * "
+"區記憶體長度。緩衝區的長度單位是 bytes，並可以用 ``array.buffer_info()[1] * "
```
Comment (mattwang44) [inline library/array.po]: Please find out the first occurrence of the term `bytes` in this article and add the translated content with it.
If `bytes（位元組）` is used for the first occurrence, use `bytes` in the rest of the article; if `位元組 (bytes)` is used for the first occurrence, use `位元組` in the rest of the article.

File: library/array.po [hunk 24]
```diff
@@ -308,9 +321,9 @@ msgid ""
 "backward compatibility and should be avoided in new code.  The buffer "
 "interface is documented in :ref:`bufferobjects`."
 msgstr ""
-"當使用來自 C 或 C++ 程式碼（這是唯一使得這個資訊有效的途徑） 的陣列物件時，更"
+"當使用來自 C 或 C++ 程式碼（這是唯一使得這個資訊有效的途徑）的陣列物件時，更"
 "適當的做法是使用陣列物件支援的緩衝區介面。這個方法維護了向後兼容性，並應該在"
-"新的程式碼中避免。關於緩衝區介面的文件在 :ref:`bufferobjects` 。"
+"新的程式碼中避免。關於緩衝區介面的文件在 :ref:`bufferobjects`\\ 。"
```
Comment (mattwang44) [inline library/array.po]: The reference of `bufferobjects` will be translated.

```suggestion
"新的程式碼中避免。關於緩衝區介面的文件在\\ :ref:`bufferobjects`\\ 。"
```

File: library/array.po [hunk 25]
```diff
@@ -335,9 +348,9 @@ msgid ""
 "iterable and its elements must be the right type to be appended to the array."
 msgstr ""
 "從 *iterable* 中新增元素到陣列的尾端，如果 *iterable* 是另一個陣列，他必須有"
-"完全相同的 type code ，如果不同會產生 :exc:`TypeError` 。如果 *iterable* 不是"
-"一個陣列，他必須可以被迭代 (iterable) 且其中的元素必須是可以被加入陣列中的正"
-"確型態。"
+"完全相同的 type code ，如果不同會導致 :exc:`TypeError`\\ 。如果 *iterable* 不"
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"完全相同的 type code，如果不同會導致 :exc:`TypeError`\\ 。如果 *iterable* 不"
```

File: library/array.po [hunk 26]
```diff
@@ -350,7 +363,7 @@ msgstr ""
 
 #: ../../library/array.rst:154
 msgid ":meth:`fromstring` is renamed to :meth:`frombytes` for clarity."
-msgstr "為了更明確，之前的 :meth:`fromstring` 被更名為 :meth:`frombytes` 。"
+msgstr "為了更明確，之前的 :meth:`fromstring` 被更名為 :meth:`frombytes`\\ 。"
```
Comment (mattwang44) [inline library/array.po]: Please also fix the translated content of `library/array.rst:219`.

```suggestion
msgstr "將 :meth:`fromstring` 更名為 :meth:`frombytes`，使其更加清晰易懂。"
```

File: library/array.po [hunk 27]
```diff
@@ -360,16 +373,16 @@ msgid ""
...
 
 #: ../../library/array.rst:168
 msgid ""
 "Append items from the list.  This is equivalent to ``for x in list: a."
 "append(x)`` except that if there is a type error, the array is unchanged."
 msgstr ""
-"從 list 中新增元素。這等價於 ``for x in list: a.append(x)`` ，除了有型態錯誤"
-"產生時，陣列會保持原狀不會被更改。"
+"從 list 中新增元素。這等價於 ``for x in list: a.append(x)``，除了有型別錯誤產"
+"生時，陣列會保持原狀不會被更改。"
```
Comment (mattwang44) [inline library/array.po:385]:
```suggestion
"生時，陣列會保持原狀。"
```

File: library/array.po [hunk 28]
```diff
@@ -378,9 +391,9 @@ msgid ""
 "``array.frombytes(unicodestring.encode(enc))`` to append Unicode data to an "
 "array of some other type."
 msgstr ""
-"用給定的 unicode 字串擴展這個陣列。陣列必須是型態 ``u`` 的陣列；其他的型態會"
-"產生 :exc:`ValueError` 錯誤。使用 ``array.frombytes(unicodestring."
-"encode(enc))`` 來新增 Unicode 資料到一個其他型態的陣列。"
+"用給定的 unicode 字串擴展這個陣列。陣列必須是 ``u`` 型別的陣列；其他的型別會"
+"導致 :exc:`ValueError` 錯誤。使用 ``array.frombytes(unicodestring."
```
Comment (mattwang44) [inline library/array.po:395]:
```suggestion
"導致 :exc:`ValueError`\\ 。使用 ``array.frombytes(unicodestring."
```

File: library/array.po [hunk 29]
```diff
@@ -389,16 +402,19 @@ msgid ""
 "specified to search for *x* within a subsection of the array.  Raise :exc:"
 "`ValueError` if *x* is not found."
 msgstr ""
+"回傳 *i* 的最小數值，使得 *i* 成為陣列之中第一次出現 *x* 的索引。選擇性的參"
```
Comment (mattwang44) [inline library/array.po]: `argument` should be translated as `引數`. Please also fix all other source strings that have this term.
```suggestion
"回傳 *i* 的最小數值，使得 *i* 成為陣列之中第一次出現 *x* 的索引。選擇性的引"
```

File: library/array.po [hunk 30]
```diff
@@ -389,16 +402,19 @@ msgid ""
...
 
 #: ../../library/array.rst:192
 msgid ""
 "Insert a new item with value *x* in the array before position *i*. Negative "
 "values are treated as being relative to the end of the array."
-msgstr "在位置 *i* 之前插入一個元素 *x* 。負數的索引值會從陣列尾端開始數。"
+msgstr "在位置 *i* 之前插入一個元素 *x*\\ 。負數的索引值會從陣列尾端開始數。"
 
 #: ../../library/array.rst:198
 msgid ""
```
Comment (mattwang44) [inline library/array.po:420]: please fix the space for library/array.rst:198.

File: library/array.po [hunk 31]
```diff
@@ -411,7 +427,7 @@ msgstr ""
 
 #: ../../library/array.rst:205
 msgid "Remove the first occurrence of *x* from the array."
-msgstr "從陣列中刪除第一個出現的 *x* 。"
+msgstr "從陣列中刪除第一個出現的 *x*\\ 。"
 
 #: ../../library/array.rst:210
 msgid "Reverse the order of the items in the array."
```
Comment (mattwang44) [inline library/array.po:433]: please help fixing the translation of library/array.rst:210

File: library/array.po [hunk 32]
```diff
@@ -428,24 +444,24 @@ msgstr ""
 
 #: ../../library/array.rst:219
 msgid ":meth:`tostring` is renamed to :meth:`tobytes` for clarity."
-msgstr "為了明確性，過去的 :meth:`tostring` 已更名為 :meth:`tobytes` 。"
+msgstr "為了明確性，過去的 :meth:`tostring` 已更名為 :meth:`tobytes`\\ 。"
 
 #: ../../library/array.rst:225
 msgid "Write all items (as machine values) to the :term:`file object` *f*."
-msgstr "將所有元素 （以 machine code 的形式）寫入 :term:`file object` *f* 。"
+msgstr "將所有元素（以 machine code 的形式）寫入 :term:`file object` *f*\\ 。"
```
Comment (mattwang44) [inline library/array.po]: why `machine value` is translated as `machine code`?

File: library/array.po [hunk 33]
```diff
@@ -428,24 +444,24 @@ msgstr ""
...
 
 #: ../../library/array.rst:235
 msgid ""
 "Convert the array to a unicode string.  The array must be a type ``'u'`` "
 "array; otherwise a :exc:`ValueError` is raised. Use ``array.tobytes()."
 "decode(enc)`` to obtain a unicode string from an array of some other type."
 msgstr ""
-"將陣列轉為一個字串。陣列的型態必須為 ``u`` 。其他型態的陣列會產生 :exc:"
-"`ValueError` 錯誤。使用 ``array.tobytes().decode(enc)`` 將其他型態的陣列轉為"
+"將陣列轉為一個字串。陣列的型態必須為 ``u`` 。其他型別的陣列會導致 :exc:"
```
Comment (mattwang44) [inline library/array.po:463]:
```suggestion
"將陣列轉為一個字串。陣列的型態必須為 ``u``，其他型別的陣列會導致 :exc:"
```

File: library/array.po [hunk 34]
```diff
@@ -428,24 +444,24 @@ msgstr ""
...
+"`ValueError` 錯誤。使用 ``array.tobytes().decode(enc)`` 將其他型別的陣列轉為"
```
Comment (mattwang44) [inline library/array.po:464]:
```suggestion
"`ValueError`\\ 。使用 ``array.tobytes().decode(enc)`` 將其他型別的陣列轉為"
```

File: library/array.po [hunk 35]
```diff
@@ -458,6 +474,13 @@ msgid ""
 "`eval`, so long as the :class:`~array.array` class has been imported using "
 "``from array import array``. Examples::"
 msgstr ""
+"當一個陣列物件被列印或轉換成字串時，他會被表示為 ``array(typecode, "
+"initializer)`` 。若為空陣列則參數 *initializer* 被省略，若 *typecode* 是 "
```
Comment (mattwang44) [inline library/array.po]:
```suggestion
"initializer)``。若為空陣列則 *initializer* 會被省略，若 *typecode* 是 "
```

File: library/array.po [hunk 36]
```diff
@@ -458,6 +474,13 @@ msgid ""
...
+"``’u’`` 將被表示為字串，其他情況則被表示為數字 list。只要 :class:`~array."
```
Comment (mattwang44) [inline library/array.po]: the term `數字 list` is too vague.

File: library/array.po [hunk 37]
```diff
@@ -458,6 +474,13 @@ msgid ""
...
+"array` class（類別）透過 ``from array import array`` 的方式引入，使用 :func:"
+"`eval` 便能確保其字串被轉換回一個擁有相同型別及數值的陣列。範例：\n"
```
Comment (mattwang44) [inline library/array.po]: `使用 eval 便能確保其字串被轉換回一個擁有相同型別及數值的陣列。` -> `便能確保該字串能夠透過 eval 來被轉換回一個擁有相同型別及數值的陣列。`

File: library/array.po [hunk 38]
```diff
@@ -485,7 +508,7 @@ msgstr "`NumPy <https://numpy.org/>`_"
 
 #: ../../library/array.rst:264
 msgid "The NumPy package defines another array type."
-msgstr "NumPy 套件定義了另一個陣列型態"
+msgstr "NumPy 套件定義了另一個陣列型別"
 
 #~ msgid ""
```
Comment (mattwang44) [inline library/array.po]: Please help delete these outdated translations at the bottom of the file.

File: library/array.po [hunk 39]
```diff
@@ -458,6 +474,13 @@ msgid ""
...
+"當一個陣列物件被列印或轉換成字串時，他會被表示為 ``array(typecode, "
+"initializer)`` 。若為空陣列則參數 *initializer* 被省略，若 *typecode* 是 "
+"``’u’`` 將被表示為字串，其他情況則被表示為數字 list。只要 :class:`~array."
+"array` class（類別）透過 ``from array import array`` 的方式引入，使用 :func:"
+"`eval` 便能確保其字串被轉換回一個擁有相同型別及數值的陣列。範例：\n"
+"\n"
+"::"
 
 #: ../../library/array.rst:257
 msgid "Module :mod:`struct`"
```
Comment (mattwang44) [inline library/array.po:486]: Should be translated as ``":mod:`struct` 模組"``. Please also fix `library/array.rst:261`.

File: library/array.po [hunk 40]
```diff
@@ -266,6 +274,11 @@ msgid ""
...
 "object>` are supported."
 msgstr ""
+"陣列支援常見的序列操作，包含索引 (indexing)、切片 (slicing)、串接 "
+"(concatenation)、相乘 (multiplication) 等。當使用切片進行賦值時，賦值的陣列必"
+"須具備相同的 type code，其他型別的數值將導致 :exc:`TypeError`\\ 。陣列同時也"
+"實作了緩衝區介面，可以在任何支援 :term:`bytes-like objects <bytes-like "
+"object>` 的地方使用。"
 
 #: ../../library/array.rst:92
 msgid "The following data items and methods are also supported:"
```
Comment (mattwang44) [inline library/array.po:284]: Replace the period in the translated sentence with `：`.

**Review by mattwang44 — CHANGES_REQUESTED** [review]

File: library/array.po [hunk 41]
```diff
@@ -360,16 +373,16 @@ msgid ""
 "inserted into the array."
 msgstr ""
 "從 :term:`file object` *f* 讀取 *n* 個 machine value 類型的元素，接著將這些元"
```
Comment (mattwang44) [inline library/array.po]: consider explaining the term `machine value`.

Comment (tomatoprinx) [reply library/array.po]: An existing translation in [tutorial/floatingpoint.po](https://github.com/python/python-docs-zh-tw/blob/3.10/tutorial/floatingpoint.po) use the word `機器數值`. I'll refer to that.

**Comment by tomatoprinx** [comment]
@mattwang44 I've create commits [b1d476d](https://github.com/python/python-docs-zh-tw/pull/140/commits/b1d476d80849e5d2487766997fa2c4cfa3eebf59) and [360dcf8](https://github.com/python/python-docs-zh-tw/pull/140/commits/360dcf8cdeda2a7de996f035b950fb395a18dbb3) to fix the previous review.

File: library/array.po [hunk 42]
```diff
@@ -243,19 +249,21 @@ msgid ""
...
 "meth:`frombytes` 或 :meth:`fromunicode` 方法（參照下方）將元素新增到其中。其"
-"他型態的變數則會傳入 :meth:`extend` 方法初始化。"
+"他情況時， 一個 iterable initializer 將被傳入 :meth:`extend` 方法之中。"
 
 #: ../../library/array.rst:80
 msgid ""
 "Raises an :ref:`auditing event <auditing>` ``array.__new__`` with arguments "
 "``typecode``, ``initializer``."
 msgstr ""
+"引發\\ :ref:`審核事件 (auditing event) <auditing>` ``array.__new__`` 並帶入引"
```
Comment (mattwang44) [inline library/array.po]: 翻譯 audit event 主頁的 #165 將其譯作「稽核事件」，這邊希望也統一用詞。

```suggestion
"引發\\ :ref:`稽核事件 (auditing event) <auditing>` ``array.__new__`` 並帶入引"
```

**Review by mattwang44 — APPROVED** [review]
Only a few minor issues remained unresolved. A new commit is added before emerging this PR. Thanks for the efforts!

**Pull request merged**